src/components/vm/disks/diskAdd.jsx

```diff
@@ -509,11 +508,39 @@ export class AddDiskModalBody extends React.Component {
             break;
         }
         case 'file': {
+            this.setState({ file: value });
+
             if (value.endsWith(".iso")) {
                 // use onValueChange instead of setState in order to perform subsequent state change logic
                 this.onValueChanged("device", "cdrom");
             }
-            this.setState({ file: value });
+
+            if (value) {
+                // Limit process time to 2 seconds and ram to 100 MB
+                cockpit.spawn(["/bin/bash", "-ec", "ulimit -t 2 -v 100000; xargs -0n1 qemu-img info --output json "], { err: "message" })
```

Comment:
This command has to use superuser permission if possible:
```suggestion
                cockpit.spawn(["/bin/bash", "-ec", "ulimit -t 2 -v 100000; xargs -0n1 qemu-img info --output json "], { err: "message", superuser: "try" })
```
Without it, if user with administrative access will try to attach a disk with a path, which requires superuser permission, e.g. `/var/lib/libvirt/images/someFile.qcow2`, the command will fail: 
`could not execute qemu-img qemu-img: Could not open '/var/lib/libvirt/images/fedora-36-beta.qcow2': Could not open '/var/lib/libvirt/images/fedora-36-beta.qcow2': Permission denied`

That makes me think, maybe there should be a test for such occassion too.